go.mod

```diff
@@ -22,7 +23,7 @@ require (
 	k8s.io/apimachinery v0.18.2
 	k8s.io/client-go v12.0.0+incompatible
 	k8s.io/kube-openapi v0.0.0-20200204173128-addea2498afe
-	kubevirt.io/client-go v0.26.0
+	kubevirt.io/client-go v0.26.2
```

Comment:
is bumping go client related to the vm importer operator?

Reply:
Yes, we require this version so the tooling updated it here as well.